AWS_LAMBDA/add_predictions_to_nonexited_guests/step_3_update_database/lambda_function.py

```diff
@@ -24,10 +21,21 @@
 predictions_file = os.environ.get('PREDICTIONS_FILE')
 predictions_file_path = '/tmp/' + predictions_file
 
+try:
+    conn = psycopg2.connect(
+        host=rds_host,
+        user=rds_username,
+        password=rds_user_pwd)
+except:
+    logger.error("ERROR: Could not connect to Postgres instance.")
+    sys.exit()
+
+logger.info("SUCCESS: Connection to RDS Postgres instance succeeded")
+
 
 def lambda_handler(event, context):
     '''Downloads and opens csv of predictions from s3 bucket, uses these to update the 
-    database table with predictions.'''
+    database table guests with predictions where predictions is null.'''
```

Comment:
As mentioned in our conversation, we will likely want to include a trigger based on a threshold percentage of their intake being complete